bench/Countdown.hs

```diff
@@ -2,7 +2,9 @@
 module Countdown where
 
 import Control.Monad.ST
+#ifdef VERSION_mtl
 import Data.Functor.Identity
```

Comment:
Please move that to the block for `mtl` imports below.

Reply:
Oh, I missed that one! Done.